src/libraries/ErrorsLib.sol

```diff
@@ -43,7 +43,7 @@ library ErrorsLib {
     error AlreadyPending();
 
     /// @notice Thrown when submitting the removal of a market when there is a cap already pending on that market.
-    error PendingCap();
+    error PendingCap(Id id);
```

Comment:
can we have both (one with and one without) ?

Reply:
Yes we could, but I guess that having only one works fine no ?